python/foglamp/services/core/api/common.py

```diff
@@ -30,20 +32,34 @@
 async def ping(request):
     """
     Args:
-        request:
+       request:
 
     Returns:
-            basic health information json payload
-            {'uptime': 32892} Time in seconds since FogLAMP started
+           basic health information json payload
+           {'uptime': 32892} Time in seconds since FogLAMP started
 
     :Example:
-            curl -X GET http://localhost:8081/foglamp/ping
+           curl -X GET http://localhost:8081/foglamp/ping
     """
     since_started = time.time() - __start_time
 
-    # TODO: FOGL-790 - ping method should return more data
-    return web.json_response({'uptime': since_started})
-
+    new_request = request.clone(rel_url='foglamp/statistics')
+    interim_stat = await get_statistics(new_request)
+    stat = json.loads(interim_stat.body.decode())
+
+    data_read = [a['value'] for a in stat if a['key'] == 'READINGS']
+    data_sent_1 = [a['value'] for a in stat if a['key'] == 'SENT_1']
+    data_sent_2 = [a['value'] for a in stat if a['key'] == 'SENT_2']
+    data_sent_3 = [a['value'] for a in stat if a['key'] == 'SENT_3']
+    data_sent_4 = [a['value'] for a in stat if a['key'] == 'SENT_4']
+    data_purged = [a['value'] for a in stat if a['key'] == 'PURGED']
```

Comment:
an internal def to this def

```
def get_stats(k):
      v = [a['value'] for a in stat if a['key'] == k]
      return v[0]
```
data_read  = get_stats('READINGS')
....

will improve readability and maintenance